evap/staff/tests/test_forms.py

```diff
@@ -257,13 +257,22 @@ def test_contribution_form_set(self):
         data['contributions-TOTAL_FORMS'] = 2
         data['contributions-1-contributor'] = user1.pk
         data['contributions-1-evaluation'] = evaluation.pk
-        data['contributions-1-questionnaires'] = questionnaire.pk
         data['contributions-1-order'] = 1
         data['contributions-1-textanswer_visibility'] = Contribution.TextAnswerVisibility.GENERAL_TEXTANSWERS
-        self.assertFalse(ContributionFormset(instance=evaluation, form_kwargs={'evaluation': evaluation}, data=data).is_valid())
+        data['contributions-1-role'] = Contribution.Role.EDITOR
+        formset = ContributionFormset(instance=evaluation, form_kwargs={'evaluation': evaluation}, data=data)
+        self.assertFalse(formset.is_valid())
+        # regression for https://github.com/e-valuation/EvaP/issues/1082
+        # assert same error message with and without questionnaire
+        self.assertEqual(formset.non_form_errors(), ['Duplicate contributor(' + contributions-1-contributor ')found. Each contributor should only be used once.'])
```

Comment:
Similar to how you created the message in the forms.py, you should use `contributor.full_name` as the string inserted into the error message. Pay attention: You forgot the second `+` and the spaces around the parentheses.